pkg/attestation/crafter/api/attestation/v1/crafting_state.proto

```diff
@@ -140,6 +140,12 @@ message PolicyEvaluation {
   // material type, if any, of the evaluated policy
   workflowcontract.v1.CraftingSchema.Material.MaterialType type = 8;
 
+  // whether this evaluation was skipped or not (because of an invalid input, for example)
+  bool skipped = 13;
+
+  // Evaluation messages, intended to communicate evaluation errors (invalid input)
+  repeated string messages = 14;
```

Comment:
should we remove messages for now, it might get confusing as we discussed the other day. i.e SARIF from different tools, telling you that it was not from that tool when in fact it went through another path?

Reply:
Moved to "skip_reason", only available when the policy is skipped